todo-facebookoauth/README.md

```diff
@@ -0,0 +1,69 @@
+# Cordova Sample App - FacebookOAuth App
```

Comment:
Consider shortening your lines to ~80-120 character limits.

Be consistent with whatever your team does.  I debate the line length limit; without them, it is difficult to read as Markdown code, but with them, it's more difficult to update.